docs/url_parameters.md

```diff
@@ -0,0 +1,25 @@
+# URL Parameters
+
+In addition to the built-in sharing feature which integrates with [Github Gists](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists), you can also manually prepare links which will pre-populate each portion of Stan Playground when clicked by using [URL Parameters](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#parameters).
```

Comment:
Given that they're already in the URL Parameters section, we can probably trim this.